content/enterprise_influxdb/v1/tools/influx_inspect.md

```diff
@@ -139,6 +142,31 @@ $ influx_inspect buildtsi -database mydb -datadir ~/.influxdb/data -waldir ~/.in
 $ influx_inspect buildtsi -database stress -shard 1 -datadir ~/.influxdb/data -waldir ~/.influxdb/wal
 ```
 
+### `check-schema`
```

Comment:
Yes, this was a convention established long ago in the 1.x CLI docs that I've always hated. At one point, I updated the InfluxDB Enterprise CLI docs to correct for this. I just never went back and updated this doc and didn't want to hang this PR up on correcting that convention. But I agree. It's not good.

While we're talking about it, there are two conventions that have always bugged me in the 1.x docs:

- The [bracketed optional influx_inspect CLI flags](https://docs.influxdata.com/influxdb/v1/tools/influx_inspect/#options) in the 1.x CLI docs. (I updated the v1 `influx` and `influxd` docs to use our "new" CLI doc conventions, but not `influx_inspect`)
- The [configuration setting headings](https://docs.influxdata.com/influxdb/v1/administration/config/#global-settings) that include the default values in the headings

Because they're headings and many of these are referenced in links, these changes have cascading affects that I didn't feel I had time to include in this PR.